File: PRAK304-2210817220011-MeldianaHandayani.py
a=int(input("Bilangan = " )) 
if a<10 and a>0:
   print("Satuan")
if a>=10 and a<20:
    print("Belasan")
if a>=20 and a<100:
    print("Puluhan")
if a==0:
    print("Nol")
elif a>=100:
    print("Anda Menginput Melebihi Limit Bilangan")

File: PRAK304-2210817220011-MeldianaHandayani.c
#include <stdio.h>
int main(){
    int a;
     printf("Masukkan Bilangan = ");
     scanf("%d", &a);
if (a<10 && a>0){
   printf("Satuan\n");
}
if (a>=10 && a<20){
    printf("Belasan\n");
}
if (a>=20 && a<100){
    printf("Puluhan\n");
}
if (a==0){
    printf("Nol\n");
}
else if (a>=100){
    printf("Anda Menginput Melebihi Limit Bilangan\n");
}
    main();
}
